config/initializers/oembed.rb

```diff
@@ -2,9 +2,11 @@
 
 OEmbed::Providers.register_all
 
-purl_provider = OEmbed::Provider.new('http://purl.stanford.edu/embed.{format}?&hide_title=true')
+#purl_provider = OEmbed::Provider.new('http://purl.stanford.edu/embed.{format}?&hide_title=true')
```

Comment:
Should this be reverted?

Reply:
Yeah the stage stuff is just for deploying to stage for users to test.

Reply:
Can you put this in a config/settings.yml and use shared config for this?